docs-src/concepts/what-is-remote-data-encoding.md

```diff
@@ -0,0 +1,36 @@
+---
+id: what-is-remote-data-encoding
+title: What is remote data encoding?
+sidebar_label: Remote data encoding
+description: Remote data encding is using your custom Data Converter to decode (and encode) your payloads remotely through endpoints.
+tags:
+  - term
+  - explanation
+---
+
+Remote data encding is using your custom Data Converter to decode (and encode) your payloads remotely through endpoints.
+
+Running your encoding remotely allows you to:
+
+- Reuse complicated encryption logic written once between different languages
+- Use it with `tctl` to encode payloads for `tctl workflow start` and with Temporal WebUI to [decode encrypted payloads](#decoding-payloads-on-the-webui-and-tctl)
+- Create a service that has access to encryption keys for performing the encryption/decryption instead of a developer workstation or service accessing the keys directly.
+
+To run data encoding/decoding remotely, use a Codec Server. A Codec Server is an HTTP server that is configured to use your custom Payload Codec with encryption/compression and decryption/decompression logic. See [Codec Server](/security#codec-server) for more information.
+
+Before you use a remote data encoder to encode your payloads, ensure that you consider all the security implications of running encryptions remotely.
```

Comment:
and more importantly "allowing decryption or decoding by performing a service call"